applications/ChimeraApplication/custom_strategies/strategies/fs_strategy_for_chimera.h

```diff
@@ -0,0 +1,836 @@
+    {
+        this->InitializeStrategy(rSolverConfig,PredictorCorrector);
+    }
+
+    /// Destructor.
+    ~FSStrategyForChimera() override{}
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    ///@}
+    ///@name Access
+    ///@{
+
+
+    ///@}
+    ///@name Inquiry
+    ///@{
+
+
+    ///@}
+    ///@name Input and output
+    ///@{
+
+    /// Turn back information as a string.
+    std::string Info() const override
+    {
+        std::stringstream buffer;
+        buffer << "FSStrategyForChimera" ;
+        return buffer.str();
+    }
+
+    /// Print information about this object.
+    void PrintInfo(std::ostream& rOStream) const override {rOStream << "FSStrategyForChimera";}
+
+    /// Print object's data.
+    void PrintData(std::ostream& rOStream) const override {}
+
+
+    ///@}
+    ///@name Friends
+    ///@{
+
+
+    ///@}
+
+protected:
+
+    ///@name Protected Life Cycle
+    ///@{
+
+
+    ///@}
+    ///@name Protected static Member Variables
+    ///@{
+
+
+    ///@}
+    ///@name Protected member Variables
+    ///@{
+
+
+    ///@}
+    ///@name Protected Operators
+    ///@{
+
+
+    ///@}
+    ///@name Protected Operations
+    ///@{
+
+
+   void SetActiveStateOnConstraint(const Flags& TheFlagToSet ,const bool ValToSet)
+    {
+        ModelPart& rModelPart = BaseType::GetModelPart();
+#pragma omp parallel
+        {
+            ModelPart::MasterSlaveConstraintIteratorType ConstraintsBegin;
+            ModelPart::MasterSlaveConstraintIteratorType ConstraintsEnd;
+            OpenMPUtils::PartitionedIterators(rModelPart.MasterSlaveConstraints(),ConstraintsBegin,ConstraintsEnd);
+
+            for ( ModelPart::MasterSlaveConstraintIteratorType itConstraint = ConstraintsBegin; itConstraint != ConstraintsEnd; ++itConstraint )
+            {
+                if (itConstraint->Is(TheFlagToSet))
+                    itConstraint->Set(ACTIVE, ValToSet);
+            }
+        }
+    }
+
+    double SolveStep() override
+    {
+
+        double start_solve_time = OpenMPUtils::GetCurrentTime();
+        ModelPart& rModelPart = BaseType::GetModelPart();
+
+        // 1. Fractional step momentum iteration
+        rModelPart.GetProcessInfo().SetValue(FRACTIONAL_STEP,1);
+
+        bool Converged = false;
+        int Rank = rModelPart.GetCommunicator().MyPID();
+
+        // Activate Constraints for VELOCITY and deactivate PRESSURE
+        SetActiveStateOnConstraint(FS_CHIMERA_VELOCITY_CONSTRAINT, true);
+        SetActiveStateOnConstraint(FS_CHIMERA_PRESSURE_CONSTRAINT, false);
+
+        for(std::size_t it = 0; it < BaseType::mMaxVelocityIter; ++it)
+        {
+            KRATOS_INFO("FRACTIONAL STEP :: ")<<it+1<<std::endl;
+            // build momentum system and solve for fractional step velocity increment
+            rModelPart.GetProcessInfo().SetValue(FRACTIONAL_STEP,1);
+            double NormDv = BaseType::mpMomentumStrategy->Solve();
```

Comment:
```suggestion
            double norm_dv = BaseType::mpMomentumStrategy->Solve();
```
Please follow variable names convention.

Reply:
done for the whole file !